test/features/grafana.feature

```diff
@@ -0,0 +1,25 @@
+Feature: Service Deployment: Grafana
+
+  Background:
+    Given Namespace is created 
+    And Kogito Operator is deployed
+    And Grafana Operator is deployed
```

Comment:
Why not reusing the step `Kogito Operator is deployed with {XXX} operator` step ?

Reply:
well, it is a thing to consider.
The original reason for creating `Kogito Operator is deployed with {XXX} operator` was that CLI was installing all depending operators in one step, so we need to adjust our steps to be same for CLI and CR approach, which is not relevant any more.

From my perspective the Grafana operator installation is completely separate action from Kogito operator installation, so it deserve separate step. That way each step is responsible on one action only.
It also helps in case of a failure, as we can better distinguish which step failed.

Reply:
oki